nextflow.config

```diff
@@ -9,36 +9,80 @@
 params {
 
   // Workflow flags
-  // TODO nf-core: Specify your pipeline's command line flags
-  reads = "data/*{1,2}.fastq.gz"
-  singleEnd = false
```

Comment:
This may be my ignorance of the method, but do you always have paired end data? Is it not possible to do single end sequencing?

Reply:
the other way around. I only(/mainly) have single end data.

Reply:
Hm ok, but can you make that assumption for others? It might be a limiting factor for the usage of your pipeline by other groups.

Reply:
It's always good to keep stuff for the 1.1 release 😉